Microsoft.HealthVault.Fhir/Transformers/PractitionerToHealthVault.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) Get Real Health.  All rights reserved.
+// MIT License
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the ""Software""), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Support;
+using Microsoft.HealthVault.Fhir.Constants;
+using Microsoft.HealthVault.ItemTypes;
+
+namespace Microsoft.HealthVault.Fhir.Transformers
+{
+    public static partial class PractitionerToHealthVault
+    {
+        public static ItemTypes.PersonItem ToHealthVault(this Hl7.Fhir.Model.Practitioner fhirPractitioner)
+        {
+            return fhirPractitioner.ToPerson();
+        }
+
+        private static ItemTypes.PersonItem ToPerson(this Hl7.Fhir.Model.Practitioner fhirPractitioner)
+        {
+            if (fhirPractitioner == null || fhirPractitioner.Name.IsNullOrEmpty())
+                return null;
+            
+            var practitionerName = fhirPractitioner.Name.First(); // Let's consider just the first item
+
+            //name
+            var person = new ItemTypes.PersonItem()
+            {
+                PersonType = new CodableValue("Provider", new CodedValue("1", "person-types")),
```

Comment:
This would make sense to live in the folder `Codes/HealthVault` where other known codes live.